src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/VSMSBuildNuGetProjectSystem.cs

```diff
@@ -481,7 +481,22 @@ where AssemblyNamesMatch(assemblyName, assemblyReferenceNode.Item2)
                     string.Format(CultureInfo.CurrentCulture, Strings.FailedToAddReference, name), e);
             }
 
-            NuGetProjectContext.Log(ProjectManagement.MessageLevel.Debug, Strings.Debug_AddReference, name, projectName);
+            NuGetProjectContext.Log(
+                ProjectManagement.MessageLevel.Debug,
+                $"Added reference '{name}' to project:'{projectName}' resolvedToPackage:{resolvedToPackage} dteOriginalPath:{dteOriginalPath} assemblyFullPath:{assemblyFullPath}.");
```

Comment:
This is a debug string and me and @emgarten discussed that it should not be localized.